packages/accounts-base/accounts_client.js

```diff
@@ -86,8 +86,29 @@ Accounts.callLoginMethod = function (options) {
     if (!options[f])
       options[f] = function () {};
   });
-  // make sure we only call the user's callback once.
-  var onceUserCallback = _.once(options.userCallback);
+  // Prepare callbacks: user provided and onLogin/onLoginFailure hooks.
+  var loginCallbacks = _.once(function (error) {
+    if (!error) {
+      onLoginHook.each(function (callback) {
+        callback({
+          allowed: true,
+          error: undefined,
```

Comment:
This is probably better as `null`.  See [the style guide](https://github.com/meteor/meteor/wiki/Meteor-Style-Guide#avoid-setting-variables-to-undefined----instead-use-null).